apps/blog/src/includes/categories.njk

```diff
@@ -1,31 +1,68 @@
-<section class="bg-secondary text-white my-4 py-4">
-  <div class="container mx-auto px-8">
-    <h2 class="text-center lg:text-left mb-4">Categories</h2>
-
-    <ul class="grid grid-cols-2 lg:grid-cols-5 gap-1 justify-items-center lg:justify-items-start subheaderUlCat">
-      {% for tag in collections.tagList | filterTagList %}
-        <li class="px-2 py-4">
-          {% set tagUrl %}/tags/{{ tag | slug }}/{% endset %}
-          <a href="{{ tagUrl | url }}" class="capitalize subheaderCat">{{ tag }}</a>
-        </li>
-      {% endfor %}
-    </ul>
-  </div>
+<section class="relative">
+  <button class="left font-bold flex items-center" id="slideLeft" type="button" name="slide-taglist-left">
+    <ion-icon name="chevron-back-outline" aria-hidden="true"></ion-icon>
+  </button>
+  <button class="right font-bold flex items-center" id="slideRight" type="button" name="slide-taglist-right">
```

Comment:
Button needs aria-label since it doesn't have text

Reply:
Updated.